packages/scripts/package.json

```diff
@@ -53,6 +55,7 @@
 		"lodash": "^4.17.15",
 		"minimist": "^1.2.0",
 		"npm-package-json-lint": "^4.0.3",
+		"prettier": "npm:wp-prettier@1.19.1",
```

Comment:
I added `prettier` as a dependency here as well. I hope it is a good idea.

Reply:
Untested, but I "think" this is correct